packages/lesswrong/lib/publicSettings.ts

```diff
@@ -0,0 +1,59 @@
+import isEmpty from 'lodash/isEmpty';
+// We initialize these public settings to make it available on both the client and the server,
+// but they get initialized via separate pathways on the client and on the server
+// Server: See databaseSettings.ts in the server directory
+// Client: See publicSettings.ts in the client directory
+export const publicSettings:Record<string, any> = {}
+
+const getNestedProperty = function (obj, desc) {
```

Comment:
This looks like a copy of a function with the same name in `packages/lesswrong/lib/vulcan-lib/utils.ts`. Better to import `Utils` and use the existing copy.

Reply:
See other comment for why that doesn't work straightforwardly. Only alternative would be to do the monkey-patching of Utils in this file, but that also seems bad for other reasons, so duplicating the code seemed overall like the best choice.

Reply:
The normal fix would be to change it from monkeypatched to real function and put it in its own file, since the new file would not have circular import issues.